### PR TITLE
[dashboard] Unify animated spinner for both light and dark themes

### PR DESCRIPTION
**Pull request by jankeromnes**
## Description

Small drive-by fix saved from https://github.com/gitpod-io/gitpod/pull/5731:

> Thanks for the [new spinner](https://github.com/gitpod-io/gitpod/issues/5374#issuecomment-921752229) @gtsiolis! Happy that it works equally well on light and dark, thus we can remove the extra theme-switching logic 😌 
> 
> | LIGHT | DARK |
> | --- | --- |
> | Screenshot 2021-09-17 at 16 07 38 | Screenshot 2021-09-17 at 16 07 58 |
> | Screenshot 2021-09-17 at 16 08 31 | Screenshot 2021-09-17 at 16 09 05 |

## Related Issue(s)

The new spinner comes from https://github.com/gitpod-io/gitpod/issues/5374#issuecomment-921752229

## How to test

1. Perform any action that shows a spinner (e.g. detect project configuration, run a prebuild...)

## Release Notes

```release-note
NONE
```

/uncc

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by:

Associated issue: *#5374*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by gtsiolis — APPROVED** [review]
Works like a charm! 🎩

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 4a722d5937362fcfe55f009efaf6baf09bc73bb4</details>

**Pull request merged**
